media/js/products/vpn/thanks-init.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+(function () {
+    'use strict';
+
+    var directDownloadLink = document.getElementById('vpn-download-link');
+    var downloadURL;
+
+    // Only auto-start the download if a supported platform is detected.
+    if (
+        Mozilla.DownloadThanks.shouldAutoDownload(window.site.platform) &&
+        typeof Mozilla.Utils !== 'undefined'
+    ) {
```

Comment:
But the download packages for each platform are different? Curious why we're allowing a Windows download on a Mac and vice versa? Is it in case someone is masking their platform?